shell/edit/provisioning.cattle.io.cluster/AddOnConfig.vue

```diff
@@ -0,0 +1,144 @@
+<script>
```

Comment:
I don't think they are very rke2 specific. One of the reasons why I wanted to do this change is because many of them I think can be reused.

Reply:
I've moved the new tabs in a separate folder, but I am not super happy with it, so I think it will need to be moved ( but right now I ma not too sure how to structure it in a way that will make more sense)